drivers/ethernet/phy_cyclonev.c

```diff
@@ -47,8 +47,8 @@
 #define PHY_RESTART_AUTONEGOTIATION BIT(9)
 
 /* Basic Status Register */
-#define PHY_BSR		      BIT(0)
-#define PHY_AUTOCAP	      BIT(30) /* Auto-negotiation capability */
+#define PHY_BSR		      (1)
```

Comment:
This looks a bit odd next to the others, if its a bit flag at bit 0 BIT(0) was correct

Reply:
Changed that, i see how it can look odd, also changed the PHY_BSR flag. PHY_CR will stay as an integer though as i would need to check what it represents